src/lib/ngsi/ParseData.h

```diff
@@ -253,14 +263,15 @@ typedef struct UpdateContextData
 *
 * UpdateContextResponseData - 
 */
-typedef struct UpdateContextResponseData
+struct UpdateContextResponseData
 {
+  UpdateContextResponseData(): cerP(0), attributeP(0), metadataP(0), domainMetadataP(0) {}
```

Comment:
Yes, all we know that NULL is 0 for the machine :) That's not the point.

The point is that *humans*, when read `param(NULL)`, tend to think "oh! that param is a pointer and it is initialized to NULL" but when read `param(0)` tend to think "oh! that param is an integer and it is initialialized to 0". Of course, nothing precludes to use this is a different way (i.e. you could use 0 in a pointer param). In this sense, this is  a kind of documentation hint, but useful anyway.